From a0714023e785791e2db77b9201db82ed8c201e62 Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Tue, 14 Apr 2015 16:10:09 +0800
Subject: [PATCH 0377/1221] MLK-10653 ARM: dts: imx7d-sdb: disable hdmi due to
 pin conflict with epdc

EPDC_DATA13 is used by HDMI and EPDC, disable HDMI for EPDC.

Signed-off-by: Robby Cai <r63905@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb-epdc.dts |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb-epdc.dts b/arch/arm/boot/dts/imx7d-sdb-epdc.dts
index 5721824..983b4f8 100644
--- a/arch/arm/boot/dts/imx7d-sdb-epdc.dts
+++ b/arch/arm/boot/dts/imx7d-sdb-epdc.dts
@@ -23,3 +23,7 @@
 &max17135 {
         status = "okay";
 };
+
+&sii902x {
+	status = "disabled";
+};
-- 
1.7.5.4

